aten/src/ATen/native/ReflectionPad.cpp

```diff
@@ -258,7 +258,12 @@ void reflection_pad2d_out_template(
   if (ndim == 3) {
     output.resize_({nplane, output_h, output_w});
   } else {
```

Comment:
Why is it enough to only update this branch?

Reply:
`resize_()` has an default argument of `memory_format=at::MemoryFormat::Contiguous`.

* for 3-dim inputs, aka. NCW, no channels last for this yet (current rule is channels last only applies to 4-dim and 5-dim tensor at the moment). Therefore we don't have to explicitly pass a memory format argument for 3-dim inputs. So even inputs are given as NWC, it will be treated as a non-contiguous NCW and getting an NCW output.
* for 4-dim inputs, aka. NCHW and 5-dim inputs aka. NCDHW, `memory_format` argument should be given as the same value of the input memory format, which is a rule of the memory format propagation: NCHW input gets a NCHW output; NHWC input gets an NHWC output.